MediaBrowser.Model/Notifications/NotificationOptions.cs

```diff
@@ -14,63 +15,53 @@ public NotificationOptions()
         {
             Options = new[]
             {
-                new NotificationOption
+                new NotificationOption(NotificationType.TaskFailed.ToString())
                 {
-                    Type = NotificationType.TaskFailed.ToString(),
                     Enabled = true,
                     SendToUserMode = SendToUserType.Admins
```

Comment:
can we somehow reduce the repetition of dozens of `enabled=true, sendtouser=admins`?

Reply:
Probably, but out of scope